frontend/src/components/Header.tsx

```diff
@@ -144,6 +144,12 @@ const HeaderNoCtx: React.FC<ContextType> = (props) => {
         users: ALL_USERS,
         exact: true
       },
+      {
+        title: 'P&E App',
```

Comment:
We should add it as a tab under "Feeds" that only shows up if the user has access.

<img width="627" alt="image" src="https://user-images.githubusercontent.com/1689183/219757369-5bfb2ac4-1cfb-4a98-beca-347e04840172.png">